client/tests/acceptance/user-can-click-package-edit-test.js

```diff
@@ -34,12 +35,18 @@ module('Acceptance | user can click package edit', function(hooks) {
     await visit('/projects');
     await click('[data-test-project="edit-pas"]');
     await click('[data-test-save-button]');
+
+    await waitFor('[data-test-submit-button]:not([disabled])');
     await click('[data-test-submit-button]');
     await click('[data-test-confirm-submit-button]');
 
     // for some reason promises aren't being captured so we await for settled state
     await settled();
 
+    // use waitFor as a way to "wait" for the transition
+    // within the pas-form/edit Component submit() task
+    await waitFor('[data-test-show-dcprevisedprojectname]');
+
```

Comment:
Cool.

Only thing that _could_ go wrong is if one of those promises (inside the task) takes a long time for some reason (uncaught network request). But we have been really good about mocking server calls.